sda-commons-server-openapi-example/src/main/java/org/sdase/commons/server/openapi/example/people/rest/Api.java

```diff
@@ -0,0 +1,64 @@
+package org.sdase.commons.server.openapi.example.people.rest;
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeIn;
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType;
+import io.swagger.v3.oas.annotations.info.Contact;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.info.License;
+import io.swagger.v3.oas.annotations.security.SecurityScheme;
+import io.swagger.v3.oas.annotations.security.SecuritySchemes;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+// Your application might have an additional place were global swagger annotations are defined, like
+// the App interface. It's only purpose is do define the annotations in the right scope.
+@OpenAPIDefinition(
+    // The following part configures the info section of the swagger documentation.
+    // Only the title as well as the resource package class needs to be configured here.
```

Comment:
The part with the resource package class seems misleading. Do you mean „This interface can be used as configured resource package class when placed in the same package as the REST API interfaces.“?

Reply:
Not sure. Just copied this file from the swagger-example.

Reply:
Hmm. I think this comment can be removed. It was previously located on the `withTitle` function of the old SwaggerBundle.Builder.